apps/README.md

```diff
@@ -189,6 +189,52 @@ fetching files served by the test runner, prefix the file path with
 document.write('<audio src="/base/test/audio/assets/win.mp3"/>');
 ```
 
+#### UI Component Style Guide ####
+
+We use `react-storybook` to generate a ui component style guide that you can use
+to discover what components are available to reuse as you build new
+features. You can also use the style guide to more easily develop new components
+without having to run all of code.org.
+
+To view the styleguide run
+
+```
+npm run storybook
```

Comment:
The npm command is `npm run` (built-in, not our own thing) which lists all the npm scripts and the commands they execute.  It's basically an alias for looking at the "scripts" entry in your package.json.